src/components/App.js

```diff
@@ -39,18 +38,30 @@ class App extends Component {
                 isLoading: false
               })
           }})
+    await fetchUsers()
+      .then(result =>{
+        if (!result.users) {
+          this.setState({
+            isLoading: false,
+            errorMsg: result
+          })
+        } else {
+          this.setState({
+              user: result.users[0],
+              isLoading: false
+            })
+        }})
```

Comment:
Looks great here, choosing there to be a user instantly instead of relying on having to log in!